spec/team_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rspec'
+require './lib/team'
+
+RSpec.describe Team do
```

Comment:
If you `require 'rspec'` as you do above, you don't need `RSpec.` below. They essentially do the same thing.

That said, does anyone have preference which one to use in all of our specs in order to be consistent? 
Let's discuss at our standup tomorrow

Reply:
Fixed requested changes